code/modules/mob/living/living.dm

```diff
@@ -279,6 +256,18 @@
 	return def_zone
 
 
+//damage/heal the mob ears and adjust the deaf amount
+/mob/living/adjustEarDamage(damage, deaf)
```

Comment:
I didn't fully implement it, just noticed it was erroring and decided it would be easier to add it and worry about implementation elsewhere later.